src/server/game/Events/GameEventMgr.cpp

```diff
@@ -533,6 +534,18 @@ void GameEventMgr::LoadFromDB()
                                          guid, newModelEquipSet.equipment_id);
                         continue;
                     }
+                    if (entry2 && !sObjectMgr->GetEquipmentInfo(entry2, equipId))
+                    {
+                        LOG_ERROR("sql.sql", "Table `game_event_model_equip` have creature (Guid: %u) with equipment_id %u not found in table `creature_equip_template`, set to no equipment.",
+                            guid, newModelEquipSet.equipment_id);
+                        continue;
+                    }
+                    if (entry3 && !sObjectMgr->GetEquipmentInfo(entry3, equipId))
+                    {
+                        LOG_ERROR("sql.sql", "Table `game_event_model_equip` have creature (Guid: %u) with equipment_id %u not found in table `creature_equip_template`, set to no equipment.",
+                            guid, newModelEquipSet.equipment_id);
+                        continue;
+                    }
```

Comment:
I think you can just combine all of these into the first if statement. 

so something like
```cpp
if (!sObjectMgr->GetEquipmentInfo(entry, equipId) || (entry2 && !sObjectMgr->GetEquipmentInfo(entry2, equipId) || (entry3 && !sObjectMgr->GetEquipmentInfo(entry3, equipId))
...
```

And then also refer to the creature id in the error log as we have multiple ids per guid :)

Reply:
Updated. Will push after compile.